src/Features/Traits/Eloquent.php

```diff
@@ -4,23 +4,147 @@
 
 namespace Blumilk\BLT\Features\Traits;
 
+use Behat\Gherkin\Node\TableNode;
+use Blumilk\BLT\LaravelRelations;
+use Illuminate\Database\Eloquent\Model;
 use Illuminate\Support\Str;
 use PHPUnit\Framework\Assert;
 
 trait Eloquent
 {
     /**
-     * @Given there is :count :model object in database
-     * @Given there should be :count :model object in database
-     * @Given there are :count :model objects in database
-     * @Given there should be :count :model objects in database
+     * @Given there is a model :model in the database:
+     * @Given there is a :model in the database:
      */
-    public function thereAreModelsInDatabase(string $model, int $count): void
+    public function seedModelInTheDatabase(string $model, ?TableNode $table = null): void
     {
         $modelClass = $this->recognizeModelClass($model);
```

Comment:
Maybe use RecognizeClassHelper instead
```suggestion
        $modelClass = RecognizeClassHelper::recognizeObjectClass($model);
```